File: static/js/conf.js
// Client config
// prod ip comes from `ip addr`
var Conf = (function() {
  var host = {
    'dev' : '0.0.0.0',
    'prod': '0.0.0.0'
  };
  var port = {
    'dev' : 1070,
    'prod': 1070
  };

  return {
    host: host,
    port: port
  };
})();

File: py/conf.py
"""
Server config
"""

DEBUG = True

HOST = {
        'DEV': '127.0.0.1',
        'PROD': '172.17.0.6' # ip exposed by Docker
       }

PORT = {
        'DEV': 8080,
        'PROD': 1070
       }

